### ISSUE TITLE
Include fixtures in separate files

### ISSUE THREAD

**Issue by DawChihLiou** [la10736/rstest#198]
Hi maintainer, thanks for the amazing crate! I'm wondering if there's a way to write fixtures in separate files. Some of my fixture sizes are vary large so I would love to put them in separate files and include them in tests. Would that be possible?

**Comment by la10736** [comment]
Yes it's possible wow fixtures in separated modules or create and then import them wherever you need

**Comment by DawChihLiou** [comment]
@la10736 thanks for getting back to me! Are there some examples I can follow? I'm not sure how to do it just by your comment. Thanks!

**Comment by la10736** [comment]
Sorry, I answered from my mobile and swipe introduced lot of craps in my answer. In https://github.com/la10736/rstest/blob/master/rstest/tests/resources/fixture/from_other_module.rs there is the example that I'm using in tests. I rewrote it here focused on reusable example

```rust
mod my_mod {
    use rstest::fixture;

    #[fixture]
    pub fn mod_fixture() -> u32 {
        42
    }
}

use my_mod::mod_fixture;
use rstest::rstest;

#[rstest]
fn example(mod_fixture: u32) {
    assert_eq!(42, mod_fixture);
}
```
